src/main/java/bio/terra/service/snapshotbuilder/query/Query.java

```diff
@@ -22,62 +20,18 @@ public record Query(
     if (tables.isEmpty()) {
       throw new IllegalArgumentException("Query must have at least one TableVariable");
     }
-    if (groupBy == null) {
-      groupBy = List.of();
-    }
-    if (orderBy == null) {
-      orderBy = List.of();
-    }
+    groupBy = groupBy != null ? groupBy : List.of();
+    orderBy = orderBy != null ? orderBy : List.of();
```

Comment:
I think the original form was fine, but if you wanted to rewrite it I would use `Objects.requireNonNullElse()` here:
```suggestion
    groupBy = Objects.requireNonNullElse(groupBy, List.of());
    orderBy = Objects.requireNonNullElse(orderBy, List.of());
```

This avoids having to duplicate the object being tested with its value in the `?:` expression.